### PR TITLE
CONTENTBOX-1011 : Error while Insert Menu widget

### PR DESCRIPTION
**Pull request by cfmitrah**
Fixed the issue

### CODE REVIEW

File: modules/contentbox/modules/contentbox-admin/views/widgets/editorSelectorHelper.cfm [hunk 1]
```diff
@@ -76,6 +76,10 @@ function switchWidgetFormMode( mode ) {
                 iconName = detail.find( "##widgetIcon" ).val();
                 titleBar.html( '<i class="fa fa-' + iconName + '"></i> Insert \'' +widgetDisplayName+ '\' Widget' );
                 //updatePreview();
+
+                if(widgetName == 'Menu' && !$('##widget-preview-content li').length){
```
Comment (lmajano) [inline modules/contentbox/modules/contentbox-admin/views/widgets/editorSelectorHelper.cfm]: Could we make this generic, where it tests ANY widget?

**Comment by cfmitrah** [comment]
@lmajano Can You Please review This and let me know if any more changes needed...

File: modules/contentbox/modules/contentbox-admin/views/widgets/editorSelectorHelper.cfm (hunk 1 above)
Comment (lmajano) [reply modules/contentbox/modules/contentbox-admin/views/widgets/editorSelectorHelper.cfm]: I would say only check for the widget preview content, so change it to only this for future proofing it:

```
if( !$('##widget-preview-content li').length ){
```

**Comment by cfmitrah** [comment]
@lmajano I have made the Changes where it will check all widgets. Sorry I got confused with another function. Can you please review and let me know if any more changes needed and Thanks for the review.

**Pull request merged**